### ISSUE TITLE
Broken Sufficient Technique links for 3.3.8

### ISSUE THREAD

**Issue by joe-watkins** [w3c/wcag#3448]
I noticed that the links to the Sufficient Techniques in 2.2 are broken. Looks like they are missing `/WAI/WCAG22/Techniques/` in their URLs.

[3.3.8 Accessible Authentication](https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication#techniques)

Link to G218  is `https://www.w3.org/techniques/general/G218` but probably should be `https://www.w3.org/WAI/WCAG22/Techniques/general/G218`

Link to H100 is `https://www.w3.org/techniques/html/H100` but probably should be `https://www.w3.org/WAI/WCAG22/Techniques/html/H100`

**Comment by fstrr** [comment]
Thanks, Joe. There were a lot of those this morning and it looked as though they'd all been fixed, but apparently not.

@iadawn Some more links need fixing.

**Comment by iadawn** [comment]
@joe-watkins Where did you bump into the [3.3.8 Accessible Authentication](https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication#techniques) link? The 3.3.8 Success Criteria links to the [Accessible Authentication (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-minimum.html) Understanding document. I believe the one you have found is to an earlier version. I will be removing this but want to check that it is not linked from elsewhere.

**Comment by joe-watkins** [comment]
@iadawn Interesting, that was from a Google search with the term "wcag 3.3.8" which the first result has `https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication` as the URL. The second result points to `https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-minimum.html`

**Comment by iadawn** [comment]
That is unfortunate. I have deleted the document, since it isn't the correct URL. Hopefully Google will take the hint.

**Comment by yatil** [comment]
@iadawn You might want to use Systeam’s redirect tool to also add a redirection. Lingering documents have always been a problem, there is probably something that can be done to clean up directories first before spooling out new content.

**Comment by patrickhlauke** [comment]
> Lingering documents have always been a problem

See also https://github.com/w3c/wcag/pull/3436 (the PR itself only removed one file from the repo, but there are still rendered versions of previous files that aren't there anymore listed in the comment)
